### PR TITLE
overlord/devicestate: make settle wait longer in remodel tests

### PR DESCRIPTION
**Pull request by bboozzoo**
We occasionally see the TestUC20RemodelSetModelWithReboot fail when running the
test suite runs on Launchpad builder on armhf or s390, but not really on other
architectures. Use a longer timeout as the test seems to be working in general,
but it is suspected that the build host/vm is so slow that settle does not
converge in time.

### CODE REVIEW

File: overlord/devicestate/devicestate_test.go [hunk 1]
```diff
@@ -248,6 +248,11 @@ func (s *deviceMgrBaseSuite) settle(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func (s *deviceMgrBaseSuite) longSettle(c *C) {
+	err := s.o.Settle(settleTimeout * 2)
```
Comment (mvo5) [inline overlord/devicestate/devicestate_test.go:252]: No strong opinion but why not make the settle timeout just longer for everything?

**Review by MiguelPires — APPROVED** [review]
LGTM

File: overlord/devicestate/devicestate_test.go (hunk 1 above)
Comment (stolowski) [reply overlord/devicestate/devicestate_test.go:252]: +1

**Review by stolowski — APPROVED** [review]
Thanks

File: overlord/devicestate/devicestate_test.go (hunk 1 above)
Comment (bboozzoo) [reply overlord/devicestate/devicestate_test.go]: This is done now.

**Pull request merged**